src/ui/Form.tsx

```diff
@@ -1,2 +1,20 @@
 import { styled } from 'stitches.config';
-export const Form = styled('form', {});
+export const Form = styled('form', {
+  variants: {
+    nominate: {
```

Comment:
In this case, rather than creating a new `Form` component, I would suggest using a regular `<form>` tag without any styles, and just adding styles to a `<Box>` that's either a parent or child of `<form>`. There's no advantage to coupling these styles to the `<form>` tag.

Reply:
@levity I am not using a new form component. I am using ui/Form that already exists should I change that?

Reply:
ah, i see. yes, i think it's ok to keep using this, but putting the styles on the tag in the NewNominateModal rather than creating the variant here.